README.md

```diff
@@ -95,11 +128,11 @@ There are 3 formatting behavior:
 - `PhoneInputBehavior.cancellable`: stop formatting when a separator is removed, do not accept non dialable chars.
 - `PhoneInputBehavior.lenient` _(default)_: stop formatting when either a non dialable char is inserted or a separator is removed.
 
-Example video: https://www.youtube.com/watch?v=rlLGVXCi-2Y.
+Example video:
 
-See `example/lib/autoformat_page.dart` for a detailed implementation.
+[![Phone number editing controller demo](http://img.youtube.com/vi/rlLGVXCi-2Y/0.jpg)](http://www.youtube.com/watch?v=rlLGVXCi-2Y "Phone number editing controller demo")
```

Comment:
I think, the static image to click and view video is a bit confusing sometimes. It would be better it there'll be the link directly like previously

Reply:
Ok. I think that a gif would be better, but can't do with this long video.
I will back to link